package.json

```diff
@@ -17,19 +17,23 @@
     "lint:es": "node ../../scripts/eslint",
     "lint:style": "node ../../scripts/stylelint",
     "lint": "yarn run lint:es && yarn run lint:style",
+    "pretest:jest_server": "node ./test/jest_integration/runIdpServer.js &",
     "test:jest_server": "node ./test/run_jest_tests.js --config ./test/jest.config.server.js",
     "test:jest_ui": "node ./test/run_jest_tests.js --config ./test/jest.config.ui.js"
   },
   "devDependencies": {
     "@elastic/eslint-import-resolver-kibana": "link:../../packages/osd-eslint-import-resolver-opensearch-dashboards",
-    "typescript": "4.0.2",
-    "gulp-rename": "2.0.0",
     "@testing-library/react-hooks": "^7.0.2",
-    "@types/hapi__wreck": "^15.0.1"
+    "@types/hapi__wreck": "^15.0.1",
+    "gulp-rename": "2.0.0",
+    "saml-idp": "^1.2.1",
+    "selenium-webdriver": "^4.0.0-alpha.7",
```

Comment:
@peterzhuamazon @bbarani Are there any concerns on building selenium into release?

Reply:
@anijain-Amazon Would it be possible to add these tests in [opensearch-dashboards-functional-test](https://github.com/opensearch-project/opensearch-dashboards-functional-test/tree/main/cypress/integration/plugins/security) and implement with Cypress? There has already been a well-automated workflow in place where developers can easily implement tests and the tests run with each build. With the workflow, developers do not have to worry about setting up clusters and configurations. They can just focus on implementing tests and the workflow takes care of everything else. In comparison, Selenium would be a brand new test framework that would require some changes in Infra for release build.

Reply:
@cliu123 currently cypress does not support saml auth workflow https://github.com/cypress-io/cypress/issues/5397, so we went with selenium

Reply:
I see. Thanks for the information!
@kavilla @peterzhuamazon Please be aware of this new test dependency Selenium.